### ISSUE TITLE
Add user-friendly labels to the Storage pulldown on the collection config page

### ISSUE THREAD

**Issue by landreev** [IQSS/dataverse#9622]
As implemented, the pulldown only shows the short names of the available stores without providing any information about their properties. 

We want to, at the very least, add the second column to the label showing the file size limit for the store. It would be helpful to also show the availability of direct download and/or upload options.

**Comment by landreev** [comment]
(only superusers get to see and use this menu, so this does not affect most curators. it will still make life easier for a few admin users).

**Comment by cmbz** [comment]
**2023/10/23:** Adding to Backlog as per @sbarbosadataverse's request and adding size 10, as per @landreev's Slack comment. Applying milestone 6.1 given urgency of the request.

**Comment by landreev** [comment]
Per earlier discussion on slack: it is unnecessary to make any code changes to achieve this. All the information in question can simply be added to the labels shown in that UI, that are defined as jvm/mpconfig settings. Something along the lines of: 
![Screen Shot 2023-10-23 at 12 41 08 PM](https://github.com/IQSS/dataverse/assets/737558/025f1b63-1aa6-495c-b0f2-c2835230f157)
Spending any time/effort on making this list look prettier would be wasteful at this point. 

I'll use the issue to make a quick doc-only PR and add some tips on making the labels informative and useful,.

**Comment by pdurbin** [comment]
I tried this quick and it worked fine:

![Screenshot 2023-11-20 at 10 38 05 AM](https://github.com/IQSS/dataverse/assets/21006/97c0b7d5-f5e3-40da-b58e-4be6c3f705e7)

```
-        -Ddataverse.files.localstack1.label=LocalStack
+        -Ddataverse.files.localstack1.label=LocalStack,direct-upload
-        -Ddataverse.files.minio1.label=MinIO
+        -Ddataverse.files.minio1.label=MinIO,non-direct-upload
```

The above comes from docker-compose-dev.yml in the S3 test PR:

- #10044

At least in that context, I couldn't figure out how to add spaces but I think that's ok. I tried `\"`. It's still readable.
